.travis.yml

```diff
@@ -21,6 +22,8 @@ matrix:
       env: SYMFONY_VERSION="2.8.*@dev"
     - php: 5.6
       env: SYMFONY_VERSION="3.0.*@dev"
```

Comment:
ok.

Do you mean that the php version is EOL or that the symfony version is EOL ?